src/assets/styles/components/_preference-survey.scss

```diff
@@ -0,0 +1,106 @@
+.preference-survey {

```

Comment:
rename service-excerpt
dont nest the modifiers name

Reply:
done